### ISSUE TITLE
Introduce the messaging panel

### ISSUE THREAD

**Issue by shilman** [storybooks/storybook#864]
<a href="https://github.com/arunoda"><img src="https://avatars2.githubusercontent.com/u/50838?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [arunoda](https://github.com/arunoda)**
_Friday Aug 12, 2016 at 07:05 GMT_
_Originally opened as https://github.com/storybooks/storybook-ui/issues/26_

----

In storybooks.io we need to show some custom messages. So we need to have a place for that in the storybook-ui.

Basically, that's to show comment notifications and related stuff. We need to find a way to set something like that using an addon.

**Comment by ndelangen** [comment]
@UsulPro Sounds something right up your alley ! 😉
